lib/fs_utils/write.js

```diff
@@ -154,4 +174,10 @@ const write = (fileList, config, joinConfig, optimizers, startTime) => {
   })).then(() => Promise.resolve({changed, disposed}));
 };
 
+const write = (fileList, config, joinConfig, optimizers, startTime) => {
+  const statics = writeStatics(fileList, config, startTime);
+  const joins = writeJoins(fileList, config, joinConfig, optimizers, startTime);
+  return statics.then(() => joins);
```

Comment:
since we now have two different flows for writing, i've split them into several functions. one for joinTo files, one for static files